templates/install/deploy-openshift.sh

```diff
@@ -178,6 +178,7 @@ do
         KEYCLOAK_PASSWORD=`random_string`
         create_self_signed_cert "oc" "standard-authservice.${NAMESPACE}.svc.cluster.local" "standard-authservice-cert"
         runcmd "oc create secret generic keycloak-credentials --from-literal=admin.username=admin --from-literal=admin.password=$KEYCLOAK_PASSWORD" "Create secret with keycloak admin credentials"
+        runcmd "oc create configmap keycloak-config --from-literal=hostname=standard-authservice --from-literal=amqpPort=5671 --from-literal=httpPort=8443 --from-literal=caSecretName=standard-authservice-cert --from-literal=oauthURL=https://keycloak-${NAMESPACE}.127.0.0.1.nip.io:443/auth" "Create standard authentication service configuration"
```

Comment:
If we create the configmap after running the template, we can get the cluster ip for the service and use that to set the hostname. Do you see any issue with that?